### ISSUE TITLE
No embed for website on Discord

### ISSUE THREAD

**Issue by sargent-coding** [ideoxan/ideoxan#72]
When I sent the link https://ideoxan.com/ on Discord, I found out that there was no embed. Maybe add this?
```html
<meta title="Ideoxan" />
    <meta charset="utf-8" />
    <meta property="og:title" content="Ideoxan" />
    <meta property="og:site_name" content="Ideoxan" />
    <meta property="og:description" content="Ideoxan is a free to use online tool to learn programming." />
    <meta
      property="og:image"
      content="https://github.com/ideoxan/brand/blob/master/exports/ix_logo_fulldark_2000x2000.png?raw=true"
    />
    <meta property="og:image:width" content="500" />
    <meta property="og:image:height" content="500" />
```

**Comment by Skyclo** [comment]
I will look into it.

**Comment by Skyclo** [comment]
Okay so in the `head.ejs` partial view (under `/content/www/views/partials/`), it has this:
```html
<meta property="og:type" 
    content="website">
<meta property="og:site_name" 
    content="<%- cfg.name %>">
<meta property="og:title" 
    content="<%- cfg.name %>: <%- (typeof title !== 'undefined')? title : 'A free to use online tool to learn programming.' %>">
<meta property="og:description" 
    content="<%- (typeof siteDescription !== 'undefined')? siteDescription : 'An interactive online learning service that offers a variety of open source courses on computer science and programming. Join today for free to start learning!' %>">
<meta property="og:image"
    content="https://ideoxan.com/static/img/ix_color_pb.png">
<meta name="theme-color"
    content="#804ddeff">
```
The variables are there so I can customize the page to whatever content is served on there. The temporary coming soon website that is currently displayed at https://ideoxan.com also has this exact same page that gets included in the served webpage. I do not believe it is the variables that are causing the issue because when viewing the webpage from the browser, it comes out exactly how it is supposed to. These variables are written when the webpage is rendered on the server, not upon page or DOM content load.

![image](https://user-images.githubusercontent.com/39171349/95002513-ea610600-05a2-11eb-8818-f88138d46f48.png)

From what I understand, the embed should work fine but I also am running into the same issue with discord. Twitter embeds, which also relies on the Open Graph protocol too, has not been working either. This means its something on our end. 

Do you know what could possibly be wrong? Is anyone else able to shed some light on the issue?

**Comment by Skyclo** [comment]
> Can I look into this?

Sure you can attempt to but it's going to be extremely difficult to test unless the IP or domain you host the site on is accessable by Discord or Twitter.

**Comment by Borumer** [comment]
Well in that case I may not be the best fit for this task 🙃

**Comment by ArduinoidIOT** [comment]
Could I be assigned to this?

**Comment by Skyclo** [comment]
> Could I be assigned to this?

Sure!

**Comment by manbonpan** [comment]
im having similar issue issue

**Comment by manbonpan** [comment]
i got my issue fixed it was my mistake

https://github.com/manbonpan/opengraphembedder_base

**Comment by Skyclo** [comment]
Fixed in aa28c3b. Closing #72.
